infra/README.md

```
@@ -22,12 +23,25 @@
 
 - Security Group
     - DB 접근용 보안 그룹 (3306 포트 오픈)
-
+    - ALB 보안 그룹: 80 포트 외부 오픈
+    - ECS Task 보안 그룹: ALB에서 들어오는 트래픽(8080) 허용
+
+- ECS & ALB
+    - ECS Cluster (uni-schedule-cluster)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)